docker/Dockerfile

```diff
@@ -0,0 +1,47 @@
+ARG PYTHON_VERSION=3.13
+ARG EM_VERSION=4.0.1
+ARG EM_NODE_VERSION=20.18.0_64bit
+ARG ARM_GNU_TOOLCHAIN_VERSION=14.2.rel1
+ARG ARM_GNU_TOOLCHAIN_PLATFORM=x86_64-arm-none-eabi
+ARG PEBBLE_BOARD_LIST="snowy_bb2 spalding_bb2 silk_bb2 asterix_evt1"
+
+FROM python:${PYTHON_VERSION} AS base
+RUN apt update && apt install -y gcc-multilib gettext clang libusb-1.0-0-dev
```

Comment:
If possible, it might be good to try to narrow exactly what gets installed here, e.g.:

```
apt install -y --no-install-suggests --no-install-recommends <every-individual-package-that-is-actually-needed>
```

Anecdotally, doing this can shave dozens or hundreds of MB off the image size, depending on exactly what all is needed.

Reply:
Added on newest commit